components/com_contact/tmpl/category/default_items.php

```diff
@@ -28,6 +28,18 @@
 $canEdit = $canDo->get('core.edit');
 $userId  = Factory::getUser()->id;
 
+$showEditColumn = false;
+if ($canEdit) {
+	$showEditColumn = true;
+} elseif ($canDo->get('core.edit.own') && !empty($this->items)) {
+	foreach ($this->items as $item) {
+		if ($item->created_by == $userId) {
+			$showEditColumn = true;
+			break;
+		}
+	}
+}		
```

Comment:
```suggestion
    $showEditColumn = true;
} elseif ($canDo->get('core.edit.own') && !empty($this->items)) {
    foreach ($this->items as $item) {
        if ($item->created_by == $userId) {
            $showEditColumn = true;
            break;
        }
    }
}
```
Fix failing code style check in drone reported here https://ci.joomla.org/joomla/joomla-cms/61738/1/6 :
- Use spaces and not tabs for indentation
- White space (tab) at the end of line 41